js/services/ConceptSet.js

```diff
@@ -62,7 +62,8 @@ define(function (require, exports) {
 	}
 
 	function loadConceptSetExpression(conceptSetId) {
-		return httpService.doGet(config.api.url + 'conceptset/' + conceptSetId + '/expression/' + sharedState.sourceKeyOfVocabUrl()).then(({ data }) => data);
+		const sourceKey = sharedState.sourceKeyOfVocabUrl();
+		return httpService.doGet(config.api.url + 'conceptset/' + conceptSetId + '/expression' + (sourceKey ? `/${sourceKey}`: '')).then(({ data }) => data);
 	}
```

Comment:
this is used to be `/expression/` but now just '/expression'. Is this correct?

Reply:
It was resolved as `/expression/null` when `sourceKey` was `null`. And that's a mistake.
If talking regarding the tail slash, I should say the `/expression` and the `/expression/` has different meanings from HTTP's point of view.